crates/cxx-qt-gen/src/naming/type_names.rs

```diff
@@ -119,6 +119,14 @@ impl TypeNames {
         type_names.populate_from_cxx_items(cxx_items, bridge_namespace, module_ident)?;
         type_names.populate_from_cxx_qt_data(cxx_qt_data, bridge_namespace, module_ident)?;
 
+        let qobject = Name {
+            rust: format_ident!("QObject"),
+            cxx: None,
+            module: None,
```

Comment:
should this not be the `module_ident` ? So that if the type is used outside the bridge it is like `ffi::QObject` ?